File: R,P,S Game.js
const choices = ['rock', 'paper', 'scissors'];

function playGame(playerChoice) {
    const computerChoice = choices[Math.floor(Math.random() * choices.length)];
    console.log("Player Choice:", playerChoice);
    console.log("Computer Choice:", computerChoice);

    if (playerChoice === computerChoice) {
        console.log("It's a tie!");
    } else if (
        (playerChoice === 'rock' && computerChoice === 'scissors') ||
        (playerChoice === 'paper' && computerChoice === 'rock') ||
        (playerChoice === 'scissors' && computerChoice === 'paper')
    ) {
        console.log("You win!");
    } else {
        console.log("You lose!");
    }
}

File: Python Practice-2025/R,P,S Game.py
import random

while True:
    print("\n1 > Rock\n2 > Paper\n3 > Scissors\n4 > Exit")
    user_input = input("Enter your choice: ")

    if user_input == "4":
        print("Exiting game. Goodbye!")
        break

    choices = {"1": "Rock", "2": "Paper", "3": "Scissors"}
    if user_input not in choices:
        print("❌ Invalid choice. Try again.")
        continue

    user_choice = choices[user_input]
    computer_choice = random.choice(list(choices.values()))

    print(f"\n🧑 You chose: {user_choice}")
    print(f"💻 Computer chose: {computer_choice}")

    if user_choice == computer_choice:
        print("🤝 It's a Tie!")
    elif (
        (user_choice == "Rock" and computer_choice == "Scissors") or
        (user_choice == "Paper" and computer_choice == "Rock") or
        (user_choice == "Scissors" and computer_choice == "Paper")
    ):
        print("✅ You Won!")
    else:
        print("💻 Computer Won!")
